src/main/kotlin/no/skatteetaten/aurora/mokey/controller/resources.kt

```diff
@@ -55,6 +55,7 @@ data class AuroraStatusResource(
 data class HealthStatusDetailResource(val code: String, val comment: String, val ref: String?)
 
 class ApplicationDeploymentDetailsResource(
+    val id: String,
```

Comment:
Jeg tror ikke dette fungerer (kommer id med ut i responsen)? Jeg mener spring-hateoas har reservert id-feltet til en link, og den kommer dermed ikke ut i json-responsen. Hvis du vil ha med id kan du bruke trikset med IdentifiedHalResource som brukes i ApplicationResource og ApplicationDeploymentResource.

Reply:
Hmm godt poeng. Kanskje vi bare heller skal kalle den applicationDeploymentId? Hva den heter er egentlig akkurat det samme for det er jo ikke i bruk enda.